### PR TITLE
Adding postLogoutRedirectUri - Sign-out with a redirect

### PR DESCRIPTION
**Pull request by Musaddiq625**
Added this:
https://learn.microsoft.com/en-us/azure/active-directory/develop/scenario-spa-sign-in?tabs=javascript2#tabpanel_4_javascript2

Without this when user logs out it redirect the user to the same page, by adding redirect URL, the developers can verify the logout process

### CODE REVIEW

**Comment by josemiguelvarela** [comment]
Thank you @Musaddiq625, I'll review it as soon as I can.

File: lib/model/config.dart [hunk 1]
```diff
@@ -182,12 +189,14 @@ class Config {
     required this.navigatorKey,
     this.origin,
     this.customParameters = const {},
+    required String postLogoutRedirectUri,
```
Comment (josemiguelvarela) [inline lib/model/config.dart:192]: @Musaddiq625 I think this parameter should be optional

**Review by josemiguelvarela — CHANGES_REQUESTED** [review]

File: lib/model/config.dart (hunk 1 above)
Comment (Musaddiq625) [reply lib/model/config.dart:192]: hm.. okay, I understand

File: assets/msalv2.js [hunk 2]
```diff
@@ -12,35 +12,44 @@ var aadOauth = (function () {
...
-  }
+ function init(config) {
+     // TODO: Add support for other MSAL configuration
+     var authData = {
+         clientId: config.clientId,
+         authority: config.isB2C ? "https://" + config.tenant + ".b2clogin.com/" + config.tenant + ".onmicrosoft.com/" + config.policy + "/" : "https://login.microsoftonline.com/" + config.tenant,
+         redirectUri: config.redirectUri,
+     };
+     var postLog = {
+         postLogoutRedirectUri: config?.postLogoutRedirectUri,
```
Comment (josemiguelvarela) [inline assets/msalv2.js:23]: Thank you for the changes @Musaddiq625. One last thing, `config` can not be null or undefined at this point, could you remove the `?` before the point.

**Review by josemiguelvarela — CHANGES_REQUESTED** [review]

**Review by josemiguelvarela — APPROVED** [review]
Looks good, thank you @Musaddiq625

**Comment by Musaddiq625** [comment]
@josemiguelvarela When will this be updated on pub.dev?
just asking, bcz I dont know the git process with pub

**Comment by josemiguelvarela** [comment]
We usually wait until we have several improvements or fixes before releasing a new version in pub.dev (this is done manually). I will discuss it with @loetsphi , but we first need to review pending pull requests.

**Comment by Musaddiq625** [comment]
got it, I'll be waiting
thanks ❤️

**Pull request merged**